docs/sources/dashboards/manage-dashboards/index.md

```diff
@@ -32,59 +32,63 @@ weight: 8
 
 # Manage dashboards
 
-A dashboard is a set of one or more [panels][] that visually presents your data in one or more rows.
-
-For more information about creating dashboards, refer to [Add and organize panels](../add-organize-panels).
-
 This topic includes techniques you can use to manage your Grafana dashboards, including:
 
 - [Creating and managing dashboard folders](#create-and-manage-dashboard-folders)
 - [Exporting and importing dashboards](#export-and-import-dashboards)
 - [Organizing dashboards](#organize-a-dashboard)
 - [Troubleshooting dashboards](#troubleshoot-dashboards)
 
+For more information about creating dashboards, refer to [Add and organize panels](../add-organize-panels).
+
+## Browse dashboards
+
+On the **Dashboards** page, you can browse and manage folders and dashboards. This includes the options to:
+
+- Create folders and dashboards
+- Move dashboards between folders
+- Delete multiple dashboards and folders
+- Navigate to a folder
+- Manage folder permissions. For more information, refer to [Dashboard permissions](https://grafana.com/docs/grafana/latest/administration/roles-and-permissions/#dashboard-permissions).
+
+{{% admonition type="note" %}}
+As of Grafana 10.2, there is no longer a special **General** folder. Dashboards without a folder are now shown at the top level alongside folders.
+{{% /admonition %}}
+
 ## Create a dashboard folder
 
 Folders help you organize and group dashboards, which is useful when you have many dashboards or multiple teams using the same Grafana instance.
 
-**Before you begin:**
-
-- Ensure that you have Grafana Admin or Super Admin permissions. For more information about dashboard permissions, refer to [Dashboard permissions][].
+> **Before you begin:** Ensure you have Editor permissions or greater to create folders. For more information about dashboard permissions, refer to [Dashboard permissions][].
 
 **To create a dashboard folder:**
 
-1. Sign in to Grafana.
-1. Click **Dashboards** in the left-side menu.
-1. On the Dashboards page, click **New** and select **New folder** in the dropdown.
+1. Click **Dashboards** in the main menu.
+1. On the **Dashboards** page, click **New** and select **New folder** in the drop-down.
 1. Enter a unique name and click **Create**.
```

Comment:
```suggestion
1. Enter a unique name and click **Create**.

  Don't use slashes (\ /) in folder names.
```

Reply:
@imatwawana I know this has been going on for a while 😅  but, I pushed up another take on this as a proper disclaimer about this restriction irt alerts. Let me know your thoughts.